src/main/java/com/marklogic/kafka/connect/sink/MarkLogicSinkConfig.java

```diff
@@ -41,6 +41,13 @@
 	public static final String DOCUMENT_MIMETYPE = "ml.document.mimeType";
 	public static final String DOCUMENT_URI_PREFIX = "ml.document.uriPrefix";
 	public static final String DOCUMENT_URI_SUFFIX = "ml.document.uriSuffix";
+	/*
+	 * v1.2.2 changes
+	 */
+	public static final String SSL = "ml.connection.ssl";
+	public static final String TLS_VERSION = "ml.connection.tlsVersion";
```

Comment:
Because these properties are specific to a custom SSL connection, I'd do "ml.connection.customSsl.tlsVersion". That makes it clear that they're not being used as part of ml.connection.simpleSsl.

Reply:
Making the changes